config/videos.json

```diff
@@ -0,0 +1,172 @@
+[
```

Comment:
Sample Data is retrieved from the API to get a preview in the Netlify deployed URL. This file and the data will be removed in the last commit.